dakota/description.md

```diff
@@ -2,61 +2,69 @@
 
 ## Introduction
```

Comment:
this name is more popular, isn't it? Ok, since this was my impression and you also say so, it is probably the less "academic" term and more appropriate.